### DISCUSSION TITLE
Thickness ?

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#230] [Q&A] [answered]
Hi  ! 
I'm wondering if it's possible to make different thickness on the same item ? 
Like with freestyle there is a way with curve ?

**Comment by pragma37** [comment, marked as answer]
Hi! Sorry for the late reply.
You can scale the line width with any value (textures, vertex colors, procedural textures...)
![imagen](https://user-images.githubusercontent.com/36610999/152252734-41a7d475-a57c-412f-93c7-7244ba5f5c68.png)
https://github.com/bnpr/Malt/discussions/126#discussioncomment-1357486

**Reply by Airioo** [reply]
Thanks !
